include/WallDistEquationSystem.h

```diff
@@ -26,7 +26,7 @@ class EquationSystems;
 class WallDistEquationSystem : public EquationSystem
 {
 public:
-  WallDistEquationSystem(EquationSystems&);
+  WallDistEquationSystem(EquationSystems&, std::string = "");
```

Comment:
Was this tested against SST? Looking at the code for default blade-resolved simulations, this will change the `ndtw` field to `minimum_distance_to_` and RANS models are expecting the field `minimum_distance_to_wall`. This will break all existing restart files.

```suggestion
  WallDistEquationSystem(EquationSystems&, std::string = "wall", bool addToEqSys = true);
```